File: cs/1.tutorial/var.cs
using System;

public class Variables
{
  static void Main(string[] args)
    {
      string firstName = "Mevia", middleName = "Jane", lastName = "Nelson";
      string fullName = firstName + ' ' + middleName + ' ' + lastName;
      int age = 16;
      float height = 180.4F;
      bool isWorking = false;
      char firstInitial = 'M', middleInitial = 'J', lastInitial = 'N';
      Console.WriteLine("My name is " + fullName + '.');
      Console.WriteLine("My intials are: " + firstInitial + middleInitial + lastInitial);
      Console.WriteLine("I am " + age + " years old.");
      Console.WriteLine("I am " + height + " cm tall!");
      Console.WriteLine("Am I working?: " + isWorking);
    }
}

File: c/1.tutorial/var.c
#include <stdio.h>

int main() {
  char firstInitial = 'M', middleInitial = 'J', lastInitial = 'N';
  int age = 16;
  float height = 180.4;
  const double PI = 3.1415926;
  printf("my initials are: %c%c%c.\n", firstInitial, middleInitial, lastInitial); // %c is the format specifier for char variables
  printf("i am %d years old.\n", age); // %d is format specifier for integers
  printf("i am %f cm tall!", height); // %f is the format specifer for floating point numbers
  printf("fun fact: the first 7 digits of pi are: %lf", PI); // %lf is the format specifier for double numbers
  return 0;
}
